seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -143,6 +143,10 @@
         {
           "text": "<code>i</code> should be a variable only defined within the for loop scope (by using<code>let</code>).",
           "testString": "getUserInput => assert(getUserInput('index').match(/(let i)/g), '<code>i</code> should be a variable only defined within the for loop scope (by using<code>let</code>).');"
+        },
+        {
+	        "text": "<code>console.log</code> should be adjusted to print the variable <code> SENTENCE </code>.",
+	        "testString": "getUserInput => assert(getUserInput('index').match(/console.log/(/s*?SENTENCE/s*?/)/s*?;/g), '<code>console.log</code> should be adjusted to print the variable <code> SENTENCE </code>.');"
```

Comment:
Also, lets change the text and testString a little bit and remove the spaces:

`"<code>console.log</code> should be changed to print the <code>SENTENCE</code> variable."`